Source/debug.cpp

```diff
@@ -565,7 +565,7 @@ std::string DebugCmdLevelUp(const std::string_view parameter)
 	Player &myPlayer = *MyPlayer;
 	std::string cmdLabel = "[givelvl] ";
 
-	const int levels = ParseInt<int>(parameter, /*min=*/1).value_or(1);
+	const int levels = ParseInt<int>(parameter, /*min=*/1, /*max=*/GetMaximumCharacterLevel()).value_or(1);

```

Comment:
This command always adds to the current player level, it'd be more correct to do something like:
```suggestion
	const int levels = ParseInt<int>(parameter, /*min=*/1, GetMaximumCharacterLevel() - myPlayer.getCharacterLevel()).value_or(1);
```

Reply:
Hm, judging by what I've read on @StephenCWills' comment, if user will enter a value above the max, let's say we have current level 1, this:
`GetMaximumCharacterLevel() - myPlayer.getCharacterLevel()` will get evaluated to 49, right? So if the user enters a number above this number, let's say 50, won't `Parseint` return `ParseIntError::outOfRange which will evaluate whole expression to 1? So it will basically clip every number to 1 instead of 50, right?

Reply:
that's a good point, using a max argument with ParseInt means you can't clamp the value. Stephen's latest comment is better.